### ISSUE TITLE
No files in the node_modules folder

### ISSUE THREAD

**Issue by warnerwarner** [flatironinstitute/labbox-ephys#450]
Hi, when I attempt to install the labbox-ephys jupyter widget I get an series of errors after giving the `pip install -e jupyterlab/labbox_ephys_widgets_jp` command. The errors all point towards files in the /labbox-ephys/jupyterlab/labbox_ephys_widgets_jp/node_modules /.stating folder. When I go to the folder there is no .staging folder there.

**Comment by magland** [comment]
Hi, could you please report your OS and version of jupyterlab? Also, version of nodejs? (node --version)

**Comment by warnerwarner** [comment]
I'm on linux, my jupyter-lab version is 3.0.9 and node is v14.15.4, thanks!

**Comment by magland** [comment]
I'm still on jupyterlab 2.x.x ... I am guessing that's the issue. If you are able to try it with jupyterlab 2 (in a conda env) that would be helpful. Of course we'd like to support jupyterlab 3, but I haven't tried it. How did you install juplab3? conda?

**Comment by warnerwarner** [comment]
So I tried a few things at once so I'm not sure what solved it exactly. I tried with jupyer-lab=2.1.2 but I was still getting the error, so I totally removed the labbox-ephys directory and then recloned it from github then tried the installation with jupyterlab=2.12 and it worked out. So I guess that maybe there was something still hanging around from the failed installation that was tripping it up.
